main_spec.tex

```diff
@@ -475,6 +475,30 @@ \subsection{Memory Ordering Routines}\label{subsec:memory_order}
 \OPR{put-with-signal}, \GET{}, and \ac{AMO} routines to symmetric data
 objects.
```

Comment:
This description is now redundant with the table. Can you shorten it similar to how you shorted the phrasing in the other changes?

Reply:
Also, there might be other places like this. It looks like you caught most of them, but you could further check by e.g., grep'ing for things like "memory store".